resources/fileTemplates/code/Magento GraphQL Resolver.php.html

```diff
@@ -6,5 +6,24 @@
 -->
```

Comment:
> Also, I renamed the method template.

I see no changes 🤔

Reply:
Sorry, I missed my commit, please try it now.
Thank you!